Makefile

```diff
@@ -33,11 +33,11 @@ LATEST_VERSION ?= NOLATEST
 # depending on GNU date.
 ISO_8601_DATE = $(shell TZ=GMT date '+%Y-%m-%dT%R:%S%z')
 
-# Sets the current Git sha
+# Sets the current Git sha.
 BUILD_SHA = $(shell git rev-parse --verify HEAD)
-# Sets the current branch
-BUILD_BRANCH = $(shell git branch --show-current)
-# Sets the current tagged git version
+# Sets the current branch.
+BUILD_BRANCH = $(shell $ git branch | awk '$$1=="*"{print $$2}')
```

Comment:
What does the additional `$` just after `shell` do here? I haven't seen that before.

Reply:
oops, that's a copy/paste error, I guess that Make swallows it

Reply:
Fixed.